crates/lib/Cargo.toml

```diff
@@ -21,6 +21,9 @@ lazy_static = "1.4.0"
 log = "0.4.17"
 num = "0.4.0"
 qcs-api = "0.2.1"
+qcs-api-client-common = "0.2.3"
+qcs-api-client-openapi = "0.3.4"
+qcs-api-client-grpc = { version = "0.2.4", features = ["server"] }
```

Comment:
The feature is only used by tests. 

Change this to `qcs-api-client-grpc = "0.2.4"`

Then under `dev-dependencies`, put

```
qcs-api-client-grpc = { version = "0.2.4", features = ["server"] }
```

You might be able to use `version = "*"`, which would avoid duplicating version numbers. Not sure if that will work correctly, though